collectors/VMPropertiesCollector.py

```diff
@@ -55,11 +57,17 @@ def do_metrics(self, target, g, i):
                         continue
                     data = value_entry['data']
                     vm_id = value_entry['resourceId']
+                    project_id = "internal"
+                    if project_ids:
+                        for pid in project_ids:
```

Comment:
`pid` is not the correct term here. You are looping through `targets` since the data structure looks like `project_id[target][vm?]['vm_id']`
I'd rather expect a `for target in project_ids:`

Reply:
No, the `for target` - loop is done in line 26 in `VMStatsCollector` respectively line 30 in `VMPropertiesCollector`. At this point, it is a list with `project ids` for every target. 

The data structure looks the following: `[{vm_id1: project_id1}, {vm_id2: project_id2}]` 

So we can talk about `for vm_id in project_ids:` That would make sense.

Reply:
Sounds good, do it.